functions/ts/src/helm_template.ts

```diff
@@ -105,6 +237,24 @@ metadata:
     config.kubernetes.io/local-config: "true"
 data:
   ${CHART_NAME}: my-chart
-  ${CHART_PATH}: ../path/to/helm/chart
-  ${VALUES}: ./values.yaml
+  ${LOCAL_CHART_PATH}: ../path/to/helm/chart
+  ${VALUES_PATH}: ./values.yaml
+
+2. To expand a chart named 'my-chart' at remote chart 'stable/chart'
```

Comment:
I would put this example first. Based on offline discussions, #1 is not a best practice for using functions.

Reply:
Will do